lampod/src/ln/inventory_manager.rs

```diff
@@ -43,6 +43,7 @@ impl InventoryHandler for LampoInventoryManager {
                     node_id: self.channel_manager.manager().get_our_node_id().to_string(),
                     peers: self.peer_manager.manager().list_peers().len(),
                     channels: self.channel_manager.manager().list_channels().len(),
+                    is_sycning: self.channel_manager.onchain.is_syncing(),
                     chain,
                     alias,
                     blockheight,
```

Comment:
Inside the inventory there if the following code 

```
                let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;
                let blockheight = height.unwrap_or_default();
```

but now this solution is not anymore a good solution because we have the `current_height` inside the chain manager

Reply:
So, rather than creating `is_syncing` method, we can directly compare the 'current_height' with the `blockheight` or we can directly display `current_height` so the user can directly see that the `current_height` is not in sync with the `blockheight`

However, if we create the `is_syncing` method, we can directly call it wherever we first need to check whether the node is still syncing or not.

Reply:
I am not talking about the logic you use inside is_syncing, but rather the GetInfo struct.

We should show best_height: blockheight and block_height: current_height.

This way, we inform the user that there is a gap between the best block and the current one.

Additionally, please remove all instances of blockheight and change them to block_height in a separate commit.

I really dislike this single-word convention; we are no longer in the '90s.